pkg/nspawntool/run.go

```diff
@@ -83,6 +84,23 @@ func RunNode(k8srelease, name string) error {
 	// defaultBinds has to be determined after evaluation of cniPath
 	defaultBinds = getDefaultBinds(cniPath)
 
+	if kubeSpawnDir == "" {
+		kubeSpawnDir = parseBind("$PWD/.kube-spawn")
+	} else if err := utils.CheckValidDir(kubeSpawnDir); err != nil {
+		kubeSpawnDir = parseBind("$PWD/.kube-spawn")
+	}
+
+	if err := bootstrap.PathSupportsOverlay(kubeSpawnDir); err != nil {
+		return fmt.Errorf("unable to create overlayfs on %q, try another filesystem like ext4.", kubeSpawnDir)
```

Comment:
"try to pass a directory with a different filesystem (like ext4 or XFS) to --kube-spawn-dir."?